code-block/src/components/ControlledCodeMirror/useFunction.tsx

```diff
@@ -0,0 +1,24 @@
+import { useCallback, useEffect, useRef } from 'react'
+
+/**
+ * Returns a function whose reference never changes.
+ *  When invoked, this function will call the function that was most recently passed to this hook.
+ *  This function is needed because the components from react-codemirror2 only recognized those functions
+ *  that were passed on the initial render.
+ * @param callback
+ */
+// eslint-disable-next-line @typescript-eslint/ban-types
+export const useFunction = <C extends undefined | Function>(
```

Comment:
What if we use less general name? Not sure what that would be.. 🤔 useUpdatedFunction?

Reply:
To be honest, whatever you put between use and Function, it should become more noticeable 😂
Feel free to change to useSyncedFunction in the next PR. I think that name is indeed clearer.